pkg/Cpanel/Security/Advisor/Assessors/Apache.pm

```diff
@@ -37,6 +37,7 @@ use Cpanel::Validate::Username ();
 use Cpanel::GenSysInfo         ();
 use Cpanel::DataStore          ();
 use Cpanel::RestartSrv         ();
+use Cpanel::KernelCare         ();
```

Comment:
Does this module exist in 62?  If not, it needs to be runtime loaded.

Reply:
Actually, it was our intention to not have this commit pinned to 62. It's really meant for 70 onward. We've discussed back porting it at some point to 62, but there are a lot of unknowns there. Please advise.

Reply:
Another note, SA/Kernel.pm has version checking for KernelCare stuff, but Apache.pm has had none. It'd get pretty nasty if we were not going to pin this commit to just 70 onward. If this is going to be an issue, then I suggest we start using a branching model rather than a single branch/multiple commit model we're currently using. In other words, if this is going to break 62, then create a 62 branch and pin 62 to it before these incompatible changes are made. This creates some tracking and backporting overhead, but this situation was inevitably going to happen.